package.json

```diff
@@ -20,7 +20,8 @@
 		"test:e2e": "wp-scripts test-e2e",
 		"test:unit": "wp-scripts test-unit-js",
 		"create-release": "node ./scripts/release.js",
-		"bump-version": "node ./scripts/commands/bump-version.js"
+		"bump-version": "node ./scripts/commands/bump-version.js",
+		"makepot": "wp i18n make-pot . languages/wc-calypso-bridge.pot && bash ./bin/update-translations.sh"
```

Comment:
I think this should follow the other build/release commands and automatically do a `git commit` if `update-translations.sh` runs successfully.

Eventually, we're going to want users to be able to run a single command which bumps the version, updates the changelog, updates the pot files, and then creates a PR for all those changes.

Reply:
Also, will `wp i18n make-pot` output a useful error message if it fails for some reason? I'd like to make sure the user gets some feedback if something goes wrong and the `update-translations.sh` step doesn't run.

Reply:
I have converted the script to node, and it has very descriptive log messages